src/axes.jl

```diff
@@ -309,15 +309,13 @@ function expand_extrema!(sp::Subplot, d::KW)
 
     # expand for bar_width
     if d[:seriestype] == :bar
-        dsym = vert ? :x : :y
+        dsym = (!vert) ? :x : :y
```

Comment:
I just reverted this change... any idea why you did this?  The idea is that it's setting a default bar width and expanding the axis extrema.  When the plot is vertical, we want to update the x axis.